compiler/scenario-service/server/src/main/scala/com/digitalasset/daml/lf/ScenarioServiceMain.scala

```diff
@@ -14,13 +20,23 @@ import io.grpc.stub.StreamObserver
 import io.grpc.{Status, StatusRuntimeException}
 import io.grpc.netty.NettyServerBuilder
 
+import scala.concurrent.ExecutionContext
+import scala.util.{Success, Failure}
 import scala.collection.concurrent.TrieMap
 import scala.collection.JavaConverters._
 import scala.util.control.NonFatal
 
 object ScenarioServiceMain extends App {
   // default to 128MB
   val maxMessageSize = args.headOption.map(_.toInt).getOrElse(128 * 1024 * 1024)
+
+  // Needed for the akka Ledger bindings used by DAML Script.
+  val system = ActorSystem("Repl")
+  implicit val sequencer: ExecutionSequencerFactory =
+    new AkkaExecutionSequencerPool("ReplPool")(system)
```

Comment:
Why the name "Repl"?

Reply:
Because I cannot remember how to spin up an actor system and copy-pasted it from the REPL :facepalm: fixed :slightly_smiling_face: